README.md

```
@@ -62,7 +62,7 @@ While we try to make sure things work also with libav, FFmpeg is preferred.
 You can use Doxygen to generate a HTML version of the API documentation:
 
     $ make docs
-    $ $BROWSER docs/html/index.html
+    $ $BROWSER doc/html/index.html
```

markdownlint-cli2 (0.23.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)